### ISSUE TITLE
Design actual UX for admin

### ISSUE THREAD

**Issue by wdoug** [codefordenver/fresh-food-connect#172]
As a developer, I want to know that the tools for the admin to use are actually valuable as to the way the program needs to be run.

**Comment by wdoug** [comment]
@jenshurley @D3s1gnerUX would either of you be willing to work on this?

**Comment by jenshurley** [comment]
Sure. One mental hurdle I run into is this: it seems that the
administrative needs during this early program-building stage are very
different from the needs for administering a program in full swing. I'm
hesitant to design for the latter-- too many unknowns. And does it need to
consider coordination of cyclists?

**Comment by jenshurley** [comment]
![sketch7322515](https://cloud.githubusercontent.com/assets/7987959/10357926/3778a2e6-6d45-11e5-9b90-77fa532c46a5.png)
![sketch73222132](https://cloud.githubusercontent.com/assets/7987959/10357931/42776cfe-6d45-11e5-8b51-7968ac4653fe.png)
![sketch73222755](https://cloud.githubusercontent.com/assets/7987959/10357936/4da7d2bc-6d45-11e5-9a87-bf9dcaeaaf0d.png)

**Comment by jenshurley** [comment]
![sketch73222936](https://cloud.githubusercontent.com/assets/7987959/10357947/6f039afe-6d45-11e5-9189-4429188c1356.png)
![sketch73223635](https://cloud.githubusercontent.com/assets/7987959/10357949/74e51826-6d45-11e5-8fe3-8fb73cbd106a.png)

Eventually admin will need to assign groups. Members of group will have the  same pickup day. 
Initially, Groups could be created by selecting individual donors from a list. When there are too many donors to select one by one, groups might be created by street where all members on selected streets are added to the group. Or they could be created from block or census block groups. Data freely available as kml) 

This is a first pass.

**Comment by lingtran** [comment]
Per conversation, we want to focus on baseline needs for this pilot season and revisit fleshing out a more comprehensive admin UX later on.  Love the donations page. Connected to card #187.

**Comment by jenshurley** [comment]
![ffc_admin_donations_superquick](https://cloud.githubusercontent.com/assets/7987959/10417983/868eed90-700b-11e5-9a86-6eb49fb586b1.jpeg)
Admin needs to be able to see at a glance who has confirmed and contact people for whom there are questions/errors/changes.
Confirmed can be yes/no if that's quickest

**Comment by D3s1gnerUX** [comment]
Sorry about my absence on this. I started a new consulting gig on Monday with Jeppesen in the DTC, and I'm still settling into the new (crazy) routine and the (1 hour each way) commute. 

This cognitive load has gotten the best of me so far.

I'll be at the meeting on Monday until 7:30ish (airport duty) and will try to be more present in general.

So much thanks to Jen for answering this call.

**Comment by lingtran** [comment]
No worries at all! 

Jen, here is Wendy/Turner's response:  This is great. One thing that Turner noticed is that someone replying “no” that they don’t have a pickup looks the same as someone just not replying at all. It would be helpful if we know who actually replied “no” vs. people who didn’t reply at all. Then we could follow up to make sure those who didn’t reply at all actually saw the email.

**Comment by jenshurley** [comment]
Makes sense! I'm thinking a gray circle or question mark for "no reply".

**Comment by jenshurley** [comment]
Started listing scenarios that affect admin's job: donor changes, admin changes, test emails, late responses, new signups, viewing routes.  [Additions to the spreadsheet would be most welcome](https://docs.google.com/spreadsheets/d/19F3p4wlHBS5kVvdSs8adDQCGUVSIywm-O_85UgeWT7M/)

**Comment by jenshurley** [comment]
![ffc_coordination_table_proposed](https://cloud.githubusercontent.com/assets/7987959/10777779/9f70a778-7ce4-11e5-8e85-aa9509a1d519.png)

![ffc-admin-send-emails](https://cloud.githubusercontent.com/assets/7987959/10777704/c16ec46e-7ce3-11e5-805b-25fb9c09525e.jpg)

The spreadsheet in previous comment contains rationale for the proposed changes to Admin Coordination Table
